docusaurus.config.js

```diff
@@ -40,11 +40,16 @@ module.exports = {
           position: 'left',
           label: 'Docs',
         },
+        {
+          href: '/classic',
+          label: 'Classic Docs',
+          position: 'right',
+        },
         {
           type: 'doc',
-          docId: 'reference/index',
-          position: 'left',
-          label: 'Reference',
+          docId: 'classic/reference/index',
+          position: 'right',
+          label: 'Classic Reference',
```

Comment:
Not sure if having these in the main navigation isn't a confusion with too much attention. Putting it into the footer could be sufficient. Plus the meaning of the word "classic" seems IMO known only internally.